test/async-test.ts

```diff
@@ -58,222 +70,230 @@ describe("AsyncMutex", () => {
-    expect(result).toBe(5);
-  });
-});
-
-describe("eventSink", () => {
-  it("receives event with no value", async () => {
-    const emitter = new EventEmitter();
-    const es = eventSink(emitter, "bingo");
-    emitter.emit("bingo");
-    expect(await es.take()).toBeUndefined();
-  });
-
-  it("receives event with single value", async () => {
-    const emitter = new EventEmitter();
-    const es = eventSink(emitter, "bingo");
-    emitter.emit("bingo", 23);
-    expect(await es.take()).toEqual(23);
-  });
-
-  it("resolves event with multiple values", async () => {
-    const emitter = new EventEmitter();
-    const es = eventSink(emitter, "bingo");
-    emitter.emit("bingo", "G", 23);
-    expect(await es.take()).toEqual(["G", 23]);
-  });
-
-  it("can receive multiple events", async () => {
-    const emitter = new EventEmitter();
-    const es = eventSink(emitter, "bingo");
-    emitter.emit("bingo", 1);
-    emitter.emit("bingo", 2);
-    expect(await es.take()).toEqual(1);
-    expect(await es.take()).toEqual(2);
-  });
-});
-
-describe("sleepAsync", () => {
-  it("sleeps", async () => {
-    const t0 = new Date().getTime();
-    await sleepAsync(110);
-    const t1 = new Date().getTime();
-    expect(t1 - t0).toBeGreaterThanOrEqual(100);
-  });
-});
-
-class TestCloseable {
-  public closed: boolean = false;
-
-  public close() {
-    this.closed = true;
-  }
-}
-
-describe("withCloseable", () => {
-  it("passes existing entity to callback", async () => {
-    const e = new TestCloseable();
-    await withCloseable(e, async (e1) => {
-      expect(e1).toBe(e);
-    });
-    expect(e.closed).toBe(true);
-  });
-
-  it("can use sync creator function", async () => {
-    const e = new TestCloseable();
-    await withCloseable(() => e, async (e1) => {
-      expect(e1).toBe(e);
-    });
-    expect(e.closed).toBe(true);
-  });
-
-  it("can use async creator function", async () => {
-    const e = new TestCloseable();
-    await withCloseable(async () => Promise.resolve(e), async (e1) => {
-      expect(e1).toBe(e);
-    });
-    expect(e.closed).toBe(true);
-  });
-
-  it("closes entity if exception is thrown", async () => {
-    const e = new TestCloseable();
-    await expect(withCloseable(e, async () => {
-      throw new Error("sorry");
-    })).rejects.toThrow();
-    expect(e.closed).toBe(true);
-  });
-
-  it("rejects if entity is null", async () => {
-    await expect(withCloseable(null, async () => true)).rejects.toThrow();
-  });
-
-  it("rejects if entity is undefined", async () => {
-    await expect(withCloseable(undefined, async () => true)).rejects.toThrow();
-  });
-
-  it("rejects if creator returns null", async () => {
-    await expect(withCloseable(() => null, async () => true)).rejects.toThrow();
-  });
-
-  it("rejects if creator returns undefined", async () => {
-    await expect(withCloseable(() => undefined, async () => true)).rejects.toThrow();
-  });
-});
+// describe("AsyncQueue", () => {
```

Comment:
I'm guessing these were only meant to be commented for local testing?

Reply:
Yikes, indeed.